uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Form/WorkPattern.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+require_once 'CRM/Core/Form.php';
+
+/**
+ * Form controller class
+ *
+ * @see http://wiki.civicrm.org/confluence/display/CRMDOC43/QuickForm+Reference
+ */
+class CRM_HRLeaveAndAbsences_Form_WorkPattern extends CRM_Core_Form
+{
+
+    /**
+     * An array used to store the loaded WorkPattern's default values, so we
+     * only need to load them once.
+     *
+     * @var array
+     */
+    private $defaultValues = [];
+
+    public function setDefaultValues() {
```

Comment:
just a final note to set doc-block for this function ... clarify that it implements CRM_CORE_FORM->setDefaultValues .